vsts.yml

```diff
@@ -57,12 +51,28 @@ phases:
       - script: |
           yarn install --force
         name: Install
-      - script: |
-          yarn lint
-        name: Lint
       - script: |
           yarn build:prod
         name: Build
       - script: |
           yarn test:setup && yarn test
         name: Test
+
+  - phase: Lint
+    queue: Hosted Linux Preview
+    steps:
+      - script: |
+          apt-get update
+          apt-get install -y --no-install-recommends libsecret-1-dev
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '8.11.1'
+      - task: geeklearningio.gl-vsts-tasks-yarn.yarn-installer-task.YarnInstaller@2
```

Comment:
Is this how Yarn is installed? If so, should that version be `1.9.4`? that's the version we use locally.

Reply:
We don't need to keep updating this because #5335 means it'll use whatever is local, but we need _a_ version of `yarn` installed, so I'm just leaving it as what the other phases are set to (macOS already has it installed).